### PR TITLE
Added action has_disk

### PR DESCRIPTION
**Pull request by zhanggbj**
Added action `has_disk` to check disk presence in SoftLayer, as described in [bosh.io](https://bosh.io/docs/cpi-api-v1.html#has-disk).

[#134980379] (https://www.pivotaltracker.com/story/show/134980379)

### CODE REVIEW

File: src/bosh-softlayer-cpi/action/has_disk.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+func NewHasDisk(
+        diskFinder DiskFinder,
+) (action HasDiskAction) {
+	action.diskFinder = diskFinder
+	return
+}
+
+func (a HasDiskAction) Run(diskCID DiskCID) (bool, error) {
+	_, found, err := a.diskFinder.Find(int(diskCID))
```
Comment (maximilien) [inline src/bosh-softlayer-cpi/action/has_disk.go]: Why not also check that the first value return is valid?

File: src/bosh-softlayer-cpi/action/has_disk_test.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+				fakeDiskFinder.FindReturns(nil, true, nil)
+			})
+
+			It("returns no error", func() {
+				Expect(err).ToNot(HaveOccurred())
+				Expect(found).To(BeTrue())
+			})
+		})
+
+		Context("when no error occurs but diskID is 0", func() {
```
Comment (maximilien) [inline src/bosh-softlayer-cpi/action/has_disk_test.go]: what about when `diskID` is invalid (e.g., negative) and not 0?

File: src/bosh-softlayer-cpi/action/has_disk.go (hunk 1 above)
Comment (zhanggbj) [inline src/bosh-softlayer-cpi/action/has_disk.go]: Done.

File: src/bosh-softlayer-cpi/action/has_disk_test.go (hunk 2 above)
Comment (zhanggbj) [inline src/bosh-softlayer-cpi/action/has_disk_test.go]: Done.

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,

I've updated PR with a new commit. Please take a review when you got time. Thanks.

**Comment by maximilien** [comment]
Looks good. Merging

**Pull request merged**